locales/en.default.schema.json

```
@@ -1781,10 +1853,7 @@
             "label": "Inverse"
           },
           "label": "Color scheme",
-          "info": "Visible when text box displayed"
-        },
-        "show_text_below": {
-          "label": "Show text below image on mobile"
+          "info": "Visible when container displayed"
```

```suggestion
          "info": "Visible when container displayed."
```
There should be a period here.

---

Same as here: https://github.com/Shopify/dawn/pull/900#discussion_r760333838